Microsoft.Toolkit.Uwp.UI.Controls/SlidableListItem/SlidableListItem.cs

```diff
@@ -245,6 +249,14 @@ private void ContentGrid_ManipulationStarted(object sender, ManipulationStartedR
                 if (_leftCommandPanel != null)
                 {
                     _leftCommandTransform = _leftCommandPanel.RenderTransform as CompositeTransform;
+
+                    _leftCommandAnimation = new DoubleAnimation();
```

Comment:
Now the animations have been updated, and I like it :-). One notable thing is that the RenderTransform need to be reset every time Stop is called, since the AnimationSet changes this when the animation is created.